Model/LogHandle.php

```diff
@@ -78,27 +78,22 @@ public function before(
         string $requestBody,
         string $requestDateTime
     ) {
-        try {
-            if ($this->config->isSecretMode()) {
-                $requestorIp = $this->secretParser->parseIp();
-                $requestHeaders = $this->secretParser->parseHeades($requestHeaders);
-                $requestBody = $this->secretParser->parseBody($requestBody);
-            }
-
-            $log = $this->logFactory->create();
-            $log->setData([
-                'request_method' => $requestMethod,
-                'requestor_ip' => $requestorIp,
-                'request_url' => $requestPath,
-                'request_headers' => $requestHeaders,
-                'request_body' => $requestBody,
-                'request_datetime' => $requestDateTime
-            ]);
-            $this->logResourceModel->save($log);
-            $this->lastLog = $log;
-        } catch (Exception $exception) {
-            $this->logger->error(__('Cant complete webapi log save because of error: %1', $exception->getMessage()));
+        if ($this->config->isSecretMode()) {
```

Comment:
Are you sure about removing this try catch? It seems like it could trigger and should be catched.